server/app/services/MessageKey.java

```diff
@@ -42,7 +42,7 @@ public enum MessageKey {
   BANNER_HTTPS_SECTION_CONTENT("banner.httpsSectionContent"),
   BANNER_LINK("banner.link"),
   BANNER_TITLE("banner.title"),
-  BANNER_VIEW_APPLICATIONS("banner.viewApplications"),
+  BANNER_VIEW_APPLICATIONS("banner.viewApplication"),
```

Comment:
suggestion:  If you're removing the "s" from the value, it should probably be removed from the key as well.

Reply:
fair. removed.